monitor/grafana/zeebe.json

```diff
@@ -20321,7 +20321,7 @@
                 "uid": "$DS_PROMETHEUS"
               },
               "editorMode": "code",
-              "expr": "sum(increase(zeebe_messaging_request_response_latency_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",partition=~\"$partition\",pod=~\"$pod\", topic=~\"atomix-membership-probe-request\"}[$__rate_interval])) by (le)",
```

Comment:
:wrench: Same as above, for the others in general. Did you double check?

Reply:
Look fine to me?
![image](https://github.com/user-attachments/assets/65fec690-f90f-46fd-b884-c9922c84102f)